JS/Mery/ui-neumorphism-mery/example/public/404.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
This file is absolute copy of the source which we gave you. YOU SHOULD NOT COPY it you should created it from scratch or delete everything which is points to sources. FIX it